### DISCUSSION TITLE
RustDesk web client V2

### DISCUSSION THREAD

**Discussion by MPalatucciEng** [rustdesk/rustdesk-server-pro#439] [Q&A]
Is RustDesk web client V2 already available for use with self-hosted PRO servers (Docker)? If so, what needs to be done to use it?

**Comment by SerWax** [comment]
It would be really nice to be able to host the web client already.

**Comment by ezueger** [comment]
+1
Any feedback if and when Pro-Customers are able to host an own web client would be apprechiated.

Thank you!

**Comment by FabianoVicente** [comment]
+1

**Comment by ConGp20** [comment]
i like the webclient and the new integration in the pro server. but i think its absolutely not ok, that you need a license with 300 clients+. I have 100 Clients and i think thats not a little installation.

**Comment by SerWax** [comment]
This is github, it's for technical discussions, not commercial complaints.
300 clients is $480 per year. If you can't afford $480 per year, you can't afford the web client. Simple as that.
Development and support for new features needs to be paid. We should be thankful it's not a separate added cost feature.

**Reply by ConGp20** [reply]
Thanks for your answer, now i got it!

**Comment by rustdesk** [comment]
You can customize web client now (RustDesk Server Pro 1.7.2)

https://github.com/rustdesk/rustdesk/wiki/FAQ#how-to-customize-self-hosted-web-client
